Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -570,8 +570,11 @@ private final class ResolvedTargetBuilder: ResolvedBuilder<ResolvedTarget> {
     }
 
     func diagnoseInvalidUseOfUnsafeFlags(_ product: ResolvedProduct) {
+        let inputTargets = product.targets
+        let recursiveDependencies = inputTargets.lazy.flatMap { $0.recursiveDependencies() }
+        let reachableTargets = Set(inputTargets).union(recursiveDependencies.compactMap { $0.target })
```

Comment:
maybe worth making this a property of product rather than a local thing

Reply:
We already have `recursiveTargetDependencies` computed property as commented above

Reply:
nice. your comment above sounded like you wanted the variable to renamed